src/styles/theme.js

```diff
@@ -87,14 +100,6 @@ export const defaultTheme = {
       messageRepliesText: {},
       image: {},
     },
-    text: {
```

Comment:
Does removing this mean we have a breaking change?

Reply:
Yeah, will publish a minor release instead of `patch`